var/spack/repos/builtin/packages/jblob/package.py

```diff
@@ -0,0 +1,37 @@
+# Copyright 2013-2019 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+import fileinput, os, shutil
+
+class Jblob(Package):
+    """The German Climate Computing Center (DKRZ: Deutsches Klimarechenzentrum GmbH) 
+       dsaprovides a Long Term Archiving Service for large research data sets which are relevant for climate or Earth system research.
+    """
+
+    homepage = "https://cera-www.dkrz.de/WDCC/ui/cerasearch"
+    url      = "http://cera-www.dkrz.de/jblob/jblob-3.0.zip"
```

Comment:
```suggestion
    url      = "https://cera-www.dkrz.de/jblob/jblob-3.0.zip"
```
Does this still work?

Reply:
Both http & https works but changed it to `https` link.